plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1204,7 +1216,7 @@ func (c *PodAffinityChecker) satisfiesPodsAffinityAntiAffinity(pod *v1.Pod, node
 		if err != nil {
 			glog.Errorf("Cannot schedule pod %+v onto node %v, because of PodAffinityTerm %v, err: %v",
 				podName(pod), node.Name, term, err)
-			return false
+			return false, PodAffinityNotMatchErr
```

Comment:
return false, nil here and cases below.

Reply:
why not return ErrPodAffinityRulesNotMatch and specific error instead of nil?